docs/development/i18n.rst

```diff
@@ -297,9 +297,16 @@ We do not want to publish the translator emails so we strip them:
        git log --pretty='%aN' $previous_version..lab/i18n -- \
         securedrop/translations install_files/ansible-base/roles/tails-config/templates | sort -u
 
+.. _i18n-administrator-permissions:
+
 Translations administrators
 ---------------------------
 
+.. note:: the translation workflow is different from the code and
+          documentation development workflow, see the associated
+          :ref:`privilege escalation <contributor-permissions>`
+          documentation.
```

Comment:
See above: any edits to the Note on the developer maintainer status should be matched by changes here.